src/Apps/ArtAdvisor/04-Bio-Follows-Alerts/lib/metaphysics.ts

```diff
@@ -0,0 +1,18 @@
+export async function metaphysics(args: {
+  query: string
+  variables: Record<string, unknown>
+  headers: Record<string, string>
+}) {
+  const { query, variables, headers } = args
+
+  const url =
+    `${process.env.METAPHYSICS_ENDPOINT}/v2` ||
```

Comment:
I sometime like to run this with our dockerized environment, especially if I want to mess with the `api` or `persistence` layers. I don't think there is really a risk of this ever pointing to prod when we're testing, but let me know if I'm not thinking of something.

Reply:
Yeah, good point. This should actually already be handled by the setup script and `.env.shared` so we should be able to delete.